src/rpc/RPC.ml

```diff
@@ -19,7 +19,7 @@ let handle_call (caps : < Cap.exec ; Cap.tmp >) :
       let modified_file_count, fixed_lines = RPC_return.autofix dryrun edits in
       Ok (`RetApplyFixes { modified_file_count; fixed_lines })
   | `CallSarifFormat
-      ( ctx,
+      ( _,
         {
           hide_nudge;
           engine_label;
```

Comment:
One question is, if the context `ctx` is no longer needed in this call, why have the parameter? 

Seems ok to leave in in the sense that it does not break something by ignoring it, and from a quick look I see that it's _some_ work to remove the parameter completely from this call.

If we don't fix that now we should open an issue to address this later. I guess this is a good compromise.
On the other hand we plan to eventually drop the python part, so... another argument against doing this work now.